wrap/run.go

```diff
@@ -45,7 +47,30 @@ func (run *Run) InitFlags() {
 	run.Cmd.StringVar(&run.EntryPoint, "entrypoint", "", "Override the default ENTRYPOINT")
 }
 
+// Checks if docker runs with user namespacing activated
+func isUserNamespaced() bool {
+	infoCmd := exec.Command("docker", "info")
+	var output bytes.Buffer
+	infoCmd.Stdout = &output
+	err := infoCmd.Run()
+	if err != nil {
+		fmt.Fprintln(os.Stderr, "Failed to execute 'docker info'")
+		os.Exit(4)
+	}
+	return bytes.Index(output.Bytes(), []byte("userns")) != -1
```

Comment:
I don't think ```userns``` could occur as a substrings anywhere else so this should be safe.
If it's part of a user or hostname it could occure even when not activated, if this is a problem we should check that no ```:``` occurs in front of it. But I don't think that that is needed right now.